microsoft-dotnet/3.1/modules/ROOT/pages/microsoft-dotnet-connector-xml-maven.adoc

```diff
@@ -0,0 +1,73 @@
+
+If you manually code a Mule runtime engine (Mule) app in XML, either from the Anypoint Studio XML editor or from a text editor, you can get access to the connector in your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
+
+When you install the connector in Studio, Studio automatically populates the XML code with the connector's namespace and schema location. It also adds a dependency to the `pom.xml` file.
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `<mule>` tag of the header of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/<connector>
+http://www.mulesoft.org/schema/mule/<connector>/current/dotnet>.xsd
+----
+
+This example shows how the namespace statements are placed in the `<mule>` XML block:
+
+[source,xml,linenums]
+----
+<mule xmlns:<connector>="http://www.mulesoft.org/schema/mule/<connector>"
+	xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:file="http://www.mulesoft.org/schema/mule/file"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core
+	http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+	http://www.mulesoft.org/schema/mule/file
+	http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd
+	http://www.mulesoft.org/schema/mule/http
+	http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+	http://www.mulesoft.org/schema/mule/ee/core
+	http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+	http://www.mulesoft.org/schema/mule/<connector>
+	http://www.mulesoft.org/schema/mule/<connector>/current/mule-dotnet.xsd">
+----
+
+== Add a POM File Dependency
+
+The Apache Maven `pom.xml` file generated by Anypoint Studio contains dependencies for a Mule app. If you code a Mule app manually, include this XML snippet in your `pom.xml` file to enable access to this connector:
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>GROUP_ID_FROM_EXCHANGE</groupId>
+  <artifactId>mule-microsoft-dotnet-connector</artifactId>
+  <version>x.x.x</version>
+  <classifier>mule-plugin</classifier>
+</dependency>
+----
+
+Replace `x.x.x` with the version that corresponds to the connector you are using.
+
+To obtain the most up-to-date `pom.xml` file information:
+
+. Go to https://www.mulesoft.com/exchange/[Anypoint Exchange].
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for `dotnet`.
+. Select the connector.
+. Click *Dependency Snippets* near the upper right of the screen.
+
+== Next Step
+
+After completing your namespace and `pom.xml` file, you can try the xref:microsoft-dotnet-connector-examples.adoc[Examples].
```

Comment:
"completing" or "providing"?

Reply:
I just noticed this....

try the Examples > try the examples, try the examples in the Examples text?